tempesta_fw/t/functional/helpers/control.py

```diff
@@ -30,6 +31,7 @@ def __init__(self, binary, uri=''):
         in `form_command()` function. This would allow to update options until
         client will be started. See `Wrk` class for example
         """
+        remote.connect_client()
```

Comment:
I assumed you still want to launch testcase modules as stand-alone programs (OTOH, I tried that and failed, there are problems with relative imports).

However — it seems like modules are re-imported afresh in each testcase, so that won't work.